src/Microsoft.Sbom.Api/Workflows/SbomParserBasedValidationWorkflow.cs

```diff
@@ -22,6 +22,7 @@
 using Microsoft.Sbom.Extensions;
 using Microsoft.Sbom.JsonAsynchronousNodeKit;
 using Microsoft.Sbom.Parser;
+using Microsoft.Sbom.Parsers.Spdx30SbomParser.ComplianceStandard;
```

Comment:
The workflow should be version-agnostic. `ElementResult` and `InvalidElementInfo` should either be moved to a more central assembly or moved behind an abstraction that is more central.

Reply:
Handled in https://github.com/microsoft/sbom-tool/pull/993